File: start.py
import random

number = [None] * 100
count = [None] * 100
countA = [None] * 100
rank = 0
ranksave = [None] * 100

#배열마다 랜덤값 부여 및 초기화
for i in range(0, 100):
	number[i] = random.randrange(1, 100)
	count[i] = 0
	
#겹치는 숫자가 몇개인지 세줌
for i in range(0, 100):
        for j in range(0, 100):
                if(number[i] == number[j]):
                        count[i] += 1
        #처음엔 무조건 더해지기에 빼줌
        count[i] -= 1

#순위를 구함
for i in range(0, 100):
        rank = 0
        #print("숫자",number[i], "갯수",count[i])
        for j in range(0, 100):
                if(count[i] < count[j]):
                        rank += 1
        ranksave[i] = rank + 1

for i in range(0, 100):
        print(i,"번","번호",number[i],"횟수",count[i],"순위",ranksave[i])

File: start.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(int argc, char *argv[]) {
	int number[100];
	int count[100];
	int countA[100];
	int i, j, rank;

	srand(time(NULL));
	for(i=0; i<100; i++){
		number[i] = rand() % 101;
		count[i] = 0;
	}
	
	for(i=0; i<100; i++){
		for(j=0; j<100; j++){
			if(number[i] == number[j]){
				count[i] ++;
			}
		}
	
		count[i] -= 1;
		printf("%d: %d\n", number[i], count[i]);
		
		for(i=0; i<100; i++)
		{
			rank = 1;
			for(j=0; j<100; j++){
				if(count[i] < count[j]){
					rank ++;
				}
				countA[i] = rank;
			}
		}
	}
	
	for(i=0; i<100; i++){
		if(countA[i] == 1){
			printf("1등은? %d번 %d번 나왔음\n", number[i], count[i]);
			break;
		}
	}

	return 0;
}